===================================================================
RCS file: /repository/php-src/ext/standard/basic_functions.c,v
retrieving revision 1.543.2.51.2.3
retrieving revision 1.543.2.51.2.4
diff -p --unified=3 -r1.543.2.51.2.3 -r1.543.2.51.2.4
--- php-src/ext/standard/basic_functions.c	2005/09/29 16:31:48	1.543.2.51.2.3
+++ php-src/ext/standard/basic_functions.c	2005/11/02 17:58:00	1.543.2.51.2.4
@@ -17,7 +17,7 @@
    +----------------------------------------------------------------------+
  */

-/* $Id$ */
+/* $Id$ */

 #include "php.h"
 #include "php_streams.h"
@@ -802,8 +802,8 @@ function_entry basic_functions[] = {
 	PHP_FE(prev,					first_arg_force_ref)
 	PHP_FE(next,					first_arg_force_ref)
 	PHP_FE(reset,					first_arg_force_ref)
-	PHP_FE(current,					NULL)
-	PHP_FE(key,					NULL)
+	PHP_FE(current,					first_arg_force_ref)
+	PHP_FE(key,					first_arg_force_ref)
 	PHP_FE(min,																NULL)
 	PHP_FE(max,																NULL)
 	PHP_FE(in_array,														NULL)
